core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -6422,11 +6422,11 @@ groups:
     - pull-ci-operator-framework-operator-lifecycle-manager-master-e2e-gcp
     - pull-ci-openshift-installer-master-e2e-aws
     - pull-ci-openshift-origin-master-e2e-gcp-builds
-    - pull-ci-openshift-odo-master-v4.3-integration-e2e-benchmark
+    - pull-ci-openshift-odo-master-v4.3-integration-e2e
     - pull-ci-openshift-origin-aggregated-logging-release-4.4-upgrade-full-integ-aws
     - pull-ci-operator-framework-operator-lifecycle-manager-master-e2e-aws-olm
-    - pull-ci-openshift-odo-master-v4.1-integration-e2e-benchmark
-    - pull-ci-openshift-odo-master-v4.2-integration-e2e-benchmark
+    - pull-ci-openshift-odo-master-v4.4-integration-e2e
```

Comment:
NIP: I guess 4.4 job is added recently. 
Moreover, 4.1 job seems to be removed.
You do not need to add new jobs into this file.
I will grep from the default cluster.

Reply:
Got it, Thanks